.github/workflows/trigger-cache-invalidation.yml

```diff
@@ -0,0 +1,26 @@
+name: Trigger cache invalidation
```

Comment:
See explanation in https://github.com/playframework/playframework/pull/11247.
Also some more notes about caching here: https://github.com/playframework/playframework/pull/11142
Right now it seems the cache gets cleared every day at around 11:10 PM.